README.md

```diff
@@ -60,3 +58,16 @@ on why you need to use `127.0.0.1`.
 ```bash
 mysql -h 127.0.0.1 -P 3306 -u deltanyc -ppassword deltanyc
 ```
+
+A note on preg UDFs
+***************************
+The cleanup utils make extensive use of [User Defined preg functions](https://github.com/mysqludf/lib_mysqludf_preg).
+These aren't available in the general MySQL docker container, and the container
+provided by the [uqlibrary](https://github.com/uqlibrary/docker-mysql-udf-preg)
+hasn't been built in a while, so it doesn't have recent fixes to the upstream
+MySQL container.
+
+One challenge here is that the `CREATE FUNCTION` statements need to be called by
+a client, as MySQL UDFs are registered in the `mysql` database. This was solved
+by creating a quick utility container (`db-udf`) that will come up with
+docker-compose, pull the relevant `CREATE` commands, run them, and exit.
```

Comment:
So, just to make sure - the db-udf container only needs to run once? (So if I just want to boot up the DB container, I should `docker compose up db db-udf`?)